audience-analytics/python-app.rst

```diff
@@ -0,0 +1,48 @@
+.. _apps:
+
+Client apps
++++++++++++
+.. after-title
+
+Our client apps are lightweight, open source scripts or executable, that you will need to install, configure and run on a your computer.
```

Comment:
Petite coquille : 'on **a your** computer'

Reply:
Lack of homogeneity on term (I think it is the same things): device, computer, player

Reply:
Fixed